ethpm_cli/_utils/filesystem.py

```diff
@@ -58,3 +61,7 @@ def atomic_replace(path: Path) -> Generator[IO[Any], None, None]:
         with tmp_file_path.open(mode="w+") as tmpfile:
             yield tmpfile
         shutil.copyfile(tmp_file_path, path)
+
+
+def is_package_installed(package_name: str, config: "Config") -> bool:
+    return os.path.exists(config.ethpm_dir / package_name)
```

Comment:
What other files or directory names might be present that this would return a false positive for?  Are there further checks  you could do about the path that would improve accuracy?  Ensure it is a directory?  Ensure it has certain files present in it?